packages/shared/providers.yaml

```diff
@@ -109,6 +109,10 @@ ashby:
     auth_mode: BASIC
     proxy:
         base_url: https://api.ashbyhq.com
+        verification:
+            method: POST
+            endpoint: apiKey.info
```

Comment:
not a big deal but what about `verification: GET /the/endpoint` in a single attribute?

Reply:
Yeah, had that thought but it seemed easier to keep it as separate endpoint / method. I remember we had that discussion with the single API endpoint work that you did a while back.